src/blockchain_utilities/blockchain_import.cpp

```diff
@@ -43,6 +43,10 @@
 #include "blockchain_db/db_types.h"
 #include "cryptonote_core/cryptonote_core.h"
 
+#ifdef _WIN32
+#include <unistd.h>
+#endif
```

Comment:
Looks like this include is necessary on Windows, because for other platforms this header seems to be included implicitly. I guess therefore this ifdef could be dropped (`sleep()` is not defined anywhere else), technically speaking.